docs/project/index.yml

```
@@ -51,15 +51,23 @@ landingContent:
             url: ../develop/develop-overview.md
 
   # Card
-  - title: Resources
+  - title: Project Developer Resources
```

I'm hesitant to change these from the generic "resources", since we're not really supporting Project-specific development at these links.
```suggestion
  - title: Resources
```